pkg/build/util/util.go

```diff
@@ -72,6 +73,30 @@ func IsPaused(bc *buildapi.BuildConfig) bool {
 	return strings.ToLower(bc.Annotations[buildapi.BuildConfigPausedAnnotation]) == "true"
 }
 
+// BuildNumber returns the given build number.
+func BuildNumber(build *buildapi.Build) (int64, error) {
+	annotations := build.GetAnnotations()
+	if stringNumber, ok := annotations[buildapi.BuildNumberAnnotation]; ok {
+		return strconv.ParseInt(stringNumber, 10, 64)
+	}
+	return 0, fmt.Errorf("build %s/%s does not have %s annotation", build.Namespace, build.Name, buildapi.BuildNumberAnnotation)
+}
+
+// BuildSchedulePolicy returns the scheduling policy for the build based on the
+// "queued" label.
+func BuildSchedulePolicy(build *buildapi.Build) buildapi.BuildSchedulePolicy {
+	labels := build.GetLabels()
+	if value, found := labels[buildapi.BuildQueuedLabel]; found {
+		switch value {
+		case "false":
+			return buildapi.BuildSchedulePolicySerialLatestOnly
+		default:
+			return buildapi.BuildSchedulePolicySerial
+		}
+	}
+	return buildapi.BuildSchedulePolicyParallel
```

Comment:
@bparees right now, only to set what policy BC had when we created this build...